### PR TITLE
Add pull request actions for linting, testing and checking the pr content

### PR DESCRIPTION
**Pull request by kaisermann**
This an [automated pull request](https://github.com/vtex/github-pull-request-script) that aims to:

- Add github pr action
- Add danger job
- Add test action job
- Add lint action job

When approved, feel free to merge it as if it was created by a bot account :robot:.

**#trivial**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+        env:
+          RUNNER_TEMP: /tmp
+      - name: Danger CI
+        uses: vtex/danger@master
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+          REQUIRE_CHANGELOG_VERSION: false
```
Comment (klzns) [inline .github/workflows/pull-request.yml:21]: I think they need this to be true?

Comment (klzns) [reply .github/workflows/pull-request.yml:21]: I'm not familiar with the release process now

**Comment by kaisermann** [comment]
Our `danger` action can check if someone has added a new version to the `changelog.md`, but I think the `styleguide` only need this when merging from the `features` branch to `master` right? cc @klzns

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (kaisermann) [reply .github/workflows/pull-request.yml:21]: See comment below

Comment (kaisermann) [reply .github/workflows/pull-request.yml:21]: Just talked to @emersonlaurentino. Only the core team is allowed to release a version, so this check will never be useful here :(

**Comment by jonathasbsouza** [comment]
@estacioneto @kaisermann what's needed for this PR to be merged?

**Comment by estacioneto** [comment]
Actually, I think it would be nice to have but only the `features` branch allows it to happen. I'm really confused about this branch and if it will be merged to `master` or not.
Maybe @matheusps or @emersonlaurentino can answer that :)

**Pull request closed without merging**